config.py

```diff
@@ -84,6 +84,15 @@
 # The file upload folder, when using models with files
 UPLOAD_FOLDER = basedir + "/app/static/uploads/"
 
+# Where to store robot code
+ROBOT_CODE_FOLDER = basedir + "/robot-code/"
+
+# Where the outputs of the simualtion are to be stored
+SIMULATION_OUTPUTS_FOLDER = basedir + "/outputs/"
+
+# The URL at which the outputs can be found/rendered
+OUTPUTS_URL = 'https://somewhere.com'
```

Comment:
Somewhere over the rainbow? Is this URL OK?

Reply:
@Adman Yeah, this will be changed in real deployment :)

The idea here is to have the outputs on a different host/subdomain than the rest of the application (mostly for security purposes).